Doc/reference/datamodel.rst

```diff
@@ -2086,11 +2086,15 @@ When a class definition is executed, the following steps occur:
 Resolving MRO entries
 ^^^^^^^^^^^^^^^^^^^^^
 
-If a base that appears in class definition is not an instance of :class:`type`,
-then an ``__mro_entries__`` method is searched on it. If found, it is called
-with the original bases tuple. This method must return a tuple of classes that
-will be used instead of this base. The tuple may be empty, in such case
-the original base is ignored.
+.. method:: object.__mro_entries__(self, bases)
+
+   If a base that appears in a class definition is not an instance of
+   :class:`type`, then an ``__mro_entries__`` method is searched on the base.
+   If an ``__mro_entries__`` method is found, the base is substituted with the
+   result of a call to ``__mro_entries__`` when creating the class.
+   The method is called with the original bases tuple, and must return a tuple
```

Comment:
This was merged before I had a chance to review, but just to note for the future, the [standard Python docs convention](https://devguide.python.org/documentation/markup/#quick-reference) is to always italicize parameter names (``*bases*``), especially on first use.

Reply:
"The original bases tuple" here refers to the object that is _passed_ to the parameter named `bases` rather than referring to the parameter named `bases` itself, so that convention doesn't apply. But if this is confusing then maybe the sentence needs to be rewarded — so this is useful feedback, thanks :)

I was planning on doing a followup PR anyway, adding links from other parts of the docs to this section in the data model. I'll see if I can address this as part of the followup PR.

Reply:
I see—I wasn't totally sure about that. In that case, then, it would be helpful, then, to explicitly mention the `bases` _parameter_ somewhere relevant.